facebook-instant-articles.php

```diff
@@ -404,4 +404,44 @@ function ia_markup_version( ) {
 	add_action( 'wp', 'ia_markup_version' );
 
 	Instant_Articles_Wizard::init();
+
+	function rescrape_article( $post_id, $post ) {
+		$adapter = new Instant_Articles_Post( $post );
+		$old_slugs = get_post_meta( $post_id, '_wp_old_slug' );
+		if ( $adapter->should_submit_post() ) {
+			try {
+				$client = Facebook\HttpClients\HttpClientsFactory::createHttpClient( null );
```

Comment:
is this now the only time we use the graph-sdk? if so, maybe we should consider either using builtin functions for a simple post or actually copying the class and dropping it as a dep?